js/scorm/wrapper.js

```diff
@@ -273,8 +273,7 @@ class ScormWrapper {
       this.setValue('cmi.learner_preference.language', lang);
       return;
     }
-    if (!this.isSupported('cmi.student_preference.language')) return;
-    this.setValue('cmi.student_preference.language', lang);
+    if (this.isChildSupported('cmi.student_preference.language')) this.setValue('cmi.student_preference.language', lang);
```

Comment:
```suggestion
    if (this.isChildSupported('cmi.student_preference.language')) {
      this.setValue('cmi.student_preference.language', lang);
    }
```
Minor: I think braces here would make this a little easier to read